packages/quantic/force-app/main/default/lwc/quanticResultPrintableUri/__tests__/quanticResultPrintableUri.test.js

```diff
@@ -0,0 +1,327 @@
+// @ts-ignore
+import * as mockHeadlessLoader from 'c/quanticHeadlessLoader';
+// @ts-ignore
+import {createElement} from 'lwc';
+import QuanticResultPrintableUri from '../quanticResultPrintableUri';
+
+const MIN_MAX_NUMBER_OF_PARTS = 3;
+
+jest.mock('c/quanticHeadlessLoader');
+
+// Helper function to mock Headless and Bueno for this suite of tests.
+function mockHeadlessAndBueno() {
+  jest.spyOn(mockHeadlessLoader, 'getHeadlessEnginePromise').mockReturnValue(
+    new Promise((resolve) => {
+      resolve();
+    })
+  );
+
+  jest.spyOn(mockHeadlessLoader, 'getBueno').mockReturnValue(
+    new Promise(() => {
+      // @ts-ignore
+      global.Bueno = {
+        isString: jest
+          .fn()
+          .mockImplementation(
+            (value) =>
+              Object.prototype.toString.call(value) === '[object String]'
+          ),
+      };
+    })
+  );
+}
+
+const selectors = {
+  link: 'c-quantic-result-link',
+  pathItem: '[data-test="path__item"]',
+  pathExpandButton: '[data-test="path__expand-button"]',
+  errorMessage: '[data-test="error"]',
+};
+const exampleEngineId = 'example engine id';
+
+function createTestComponent(options) {
+  const element = createElement('c-quantic-result-printable-uri', {
+    is: QuanticResultPrintableUri,
+  });
+
+  for (const [key, value] of Object.entries(options)) {
+    element[key] = value;
+  }
+
+  document.body.appendChild(element);
+  return element;
+}
+
+// Helper function to wait until the microtask queue is empty.
+function flushPromises() {
+  return new Promise((resolve) => setTimeout(resolve, 0));
+}
+
+function generateExampleUri(id) {
+  return `https://uri${id}.com/`;
+}
+
+function generateResultParents(numberOfParents) {
+  let parents = '';
+  for (let i = 0; i < numberOfParents; i++) {
+    parents += `<parent name="parent${i + 1}" uri="${generateExampleUri(i + 1)}" />`;
+  }
```

Comment:
```suggestion
  for (let i = 1; i <= numberOfParents; i++) {
    parents += `<parent name="parent${i}" uri="https://uri${i}.com/" />`;
  }
```
Goes for everywhere else if it repeats

Reply:
Ah I see after looking at the tests why you would need this xD